be/src/exec/pipeline/scan/connector_scan_operator.cpp

```diff
@@ -24,6 +24,12 @@
 namespace starrocks::pipeline {
 
 // ==================== ConnectorScanOperatorFactory ====================
+ConnectorScanOperatorMemShareArbitrator::ConnectorScanOperatorMemShareArbitrator(int64_t query_mem_limit,
+                                                                                 int scan_node_number)
+        : query_mem_limit(query_mem_limit),
+          scan_mem_limit(query_mem_limit),
```

Comment:
Why remove `scan_mem_limit = query_mem_limit *  connector_scan_use_query_mem_ratio` ?

Reply:
Because we don't want scan_mem_limit limited by be.conf.  Instead it should only be limited by fe, which is more flexible.